resources/home/dnanexus/Pan_Cancer_Sample_Report.Rmd

```diff
@@ -0,0 +1,511 @@
+* coverage_CV = coefficient of variation of coverage for each gene (relative measre of dispersion)
+
+```{r coverage}
+rnaseqc_coverage <- read.csv(params$qcgenecov, sep = "\t")
+# get the genes on the capture
+capture_genes <- read.csv(params$capturegenes, header = F)
+rnaseqc_coverage_hgnc <- rnaseqc_coverage[which(rnaseqc_coverage$hgnc_symbol %in% capture_genes$V1),]
+max_cov <- as.numeric(rnaseqc_coverage_hgnc[order(rnaseqc_coverage_hgnc$coverage_mean, decreasing = T),][1,"coverage_mean"])
+max_gene <-  rnaseqc_coverage_hgnc[order(rnaseqc_coverage_hgnc$coverage_mean, decreasing = T),][1,"hgnc_symbol"]                 
+```
+
+
+```{r}
+genes <- genes <- unlist(strsplit(fusion_abdriged$X.FusionName, "::"))
+rnaseqc_coverage_hgnc_fusions <- rnaseqc_coverage_hgnc[which(rnaseqc_coverage_hgnc$hgnc_symbol %in% genes),]
+
+
+rnaseqc_coverage_hgnc %>%
+  ggplot(aes(rnaseqc_coverage_hgnc$coverage_mean)) +
+  geom_boxplot() +
+  geom_dotplot(data = rnaseqc_coverage_hgnc_fusions,
+               aes(rnaseqc_coverage_hgnc_fusions$coverage_mean),binaxis='x',
+               stackdir='center',dotsize=0.5,color='red') +
+  theme(
+         panel.background = element_rect(fill='transparent'),
+         plot.background = element_rect(fill='transparent', color=NA),
+         panel.grid.major = element_blank(),
+         panel.grid.minor = element_blank(),
+         legend.background = element_rect(fill='transparent'),
+         legend.box.background = element_rect(fill='transparent')
+       ) +
+  labs(x = "Mean Coverage", title = "All genes")
+```
+
+```{r}
+datatable(rnaseqc_coverage_hgnc, rownames = F, 
+  escape=FALSE, extensions = c('FixedColumns','Buttons'),
+  options = list(
+    scrollX = TRUE,
+    fixedColumns = list(leftColumns = 1),
+    dom = 'Bfrtip',
+    buttons = c('copy', 'csv', 'excel')
+  )
+)
+```
+
+
+### 3.2 Coverage by Exon
+
+```{r econ_coverage}
+rnaseqc_exons <- read.csv(params$qcexonscov, sep = "\t")
+rnaseqc_exons <- separate(data = rnaseqc_exons, col = Exon.ID, into = c("Gene_ID", "Exon"), sep = "_")
+# get the genes on the capture
+rnaseqc_exonse_hgnc <- rnaseqc_exons[which(rnaseqc_exons$Gene_ID %in% rnaseqc_coverage_hgnc$ENSG_id),]
+rnaseqc_exonse_hgnc$hgnc_symbol <- rnaseqc_coverage_hgnc$hgnc_symbol[match(rnaseqc_exonse_hgnc$Gene_ID, rnaseqc_coverage_hgnc$ENSG_id)]
+rnaseqc_exonse_hgnc$hgnc_id<- rnaseqc_coverage_hgnc$hgnc_id[match(rnaseqc_exonse_hgnc$Gene_ID, rnaseqc_coverage_hgnc$ENSG_id)]
+rnaseqc_exonse_hgnc <- rnaseqc_exonse_hgnc[,c("hgnc_symbol", "hgnc_id", "Gene_ID", "Exon", "Exon.CV")]
+```
+
+```{r}
+datatable(rnaseqc_exonse_hgnc, rownames = F, 
+  escape=FALSE, extensions = c('FixedColumns','Buttons'),
+  options = list(
+    scrollX = TRUE,
+    fixedColumns = list(leftColumns = 1),
+    dom = 'Bfrtip',
+    buttons = c('copy', 'csv', 'excel')
+  )
+)
+```
+
+
+## 4. Session Info
+
+```{r 4_session_info, include = TRUE}
+
+sessionInfo()
+
+```
```

Comment:
This is not the latest/full code. Pls. share the latest/full code.
Pls. add a References section.